src/set_theory/ordinal_arithmetic.lean

```diff
@@ -743,6 +743,10 @@ begin
   { exact mul_is_limit l.pos lb }
 end
 
+theorem smul_eq_mul {a : ordinal} : ∀ {n : ℕ}, n • a = a * n
```

Comment:
Usually this would be stated
```suggestion
theorem smul_eq_mul {a : ordinal} : ∀ {n : ℕ}, n • a = n * a
```

Reply:
Be careful! Ordinal multiplication isn't commutative. In fact, the statement as you worded it is false: `2 • ω = ω * 2 ≠ 2 * ω`.